projects/unix/Makefile

```diff
@@ -439,7 +440,7 @@ SOURCE = \
 	$(SRCDIR)/pi/pi_controller.c \
 	$(SRCDIR)/pi/sram.c \
 	$(SRCDIR)/plugin/emulate_game_controller_via_input_plugin.c \
-	$(SRCDIR)/plugin/emulate_speaker_via_audio_plugin.c \
+	$(SRCDIR)/plugin/audio_backend_compat.c \
```

Comment:
I put the audio_backend_compat in the plugin folder because it is related to plugins, and will also go away when plugins are dropped.